packages/typespec-azure-resource-manager/lib/models.tsp

```diff
@@ -302,6 +302,25 @@ model EntityTagProperty {
   eTag?: string;
 }
 
+/**
+ * Model used only to spread in the standard `etag` envelope property for a resource
+ *
+ * @example
+ *
+ * ```typespec
+ * model Foo is TrackedResource<FooProperties> {
+ *   // Only have standard Succeeded, Failed, Cancelled states
+ *   ...ETagProperty;
+ * }
+ * ```
+ */
+@doc("The etag property envelope.")
```

Comment:
I think this doc is unnecessary as it should always be spread which make me think maybe we should make those aliases of model expr instead